db/seeds.rb

```diff
@@ -132,3 +141,42 @@
 
   user.organisations << organisation
 end
+
+access_requester_user = User.create!(
+  first_name: 'Jane',
```

Comment:
Can we use Faker? i.e. `Faker::Name.unique.name`

Reply:
good idea, will do